jambo.json

```diff
@@ -15,4 +15,4 @@
     "preservedFiles": []
   },
   "defaultTheme": "answers-hitchhiker-theme"
-}
+}
```

Comment:
nit: can we add back the ending newline?

Reply:
updated!